availability-prober/availability_prober.go

```diff
@@ -187,6 +190,15 @@ func check(log logr.Logger, target *url.URL, requestTimeout time.Duration, sleep
 			}
 		}
 
+		if len(waitForClusterRolebinding) > 0 {
+			clusterRoleBinding := &rbacv1.ClusterRoleBinding{}
+			err := kubeClient.Get(context.Background(), types.NamespacedName{Name: waitForClusterRolebinding}, clusterRoleBinding)
+			if err != nil {
+				log.Error(err, "failed to get cluster rolebinding, retrying...")
```

Comment:
this is not an error, just log it as an Info. It would also be good to know which clusterrolebinding you are waiting for